src/btree/bt_delete.c

```diff
@@ -331,9 +328,9 @@ __tombstone_update_alloc(WT_SESSION_IMPL *session,
 	 * timestamp, do nothing.
 	 */
 	if (page_del != NULL) {
-		upd->txnid = page_del->txnid;
-		upd->prepare_state = page_del->prepare_state;
 		__wt_timestamp_set(&upd->timestamp, &page_del->timestamp);
+		upd->prepare_state = page_del->prepare_state;
+		upd->txnid = page_del->txnid;
```

Comment:
I'm not sure about this change -- I'm guessing this was just a cosmetic change that fell out of some commit, in which case, it's fine with me. If it's more than cosmetic, then I don't understand it.

In general, I like initializing structures in the order the fields are declared simply because it makes it easier to figure out if any fields were forgotten.